### ISSUE TITLE
wrong integer comparison throws an exception

### ISSUE THREAD

**Issue by maxadamo** [matteocorti/check_ssl_cert#153]
**Describe the bug**\
I ran this command:
```
./check_ssl_cert -H tools.geant.net -L 'C' --check-ssl-labs-warn-grade 'B'
```
and I have got this error:
```
./check_ssl_cert: 1287: /opt/sensu-plugins-ruby/embedded/bin/check_ssl_cert: cannot open 65: No such file
./check_ssl_cert: 1287: /opt/sensu-plugins-ruby/embedded/bin/check_ssl_cert: 65: not found
```

**Comment by maxadamo** [comment]
too lazy to make a PR for this. Sorry :smiley:  But if you usethe statement below on line 1287 it works: 
```
if [ "${SSL_LAB_WARN_ASSESTMENT_NUMERIC}" -lt "${SSL_LAB_CRIT_ASSESSMENT_NUMERIC}" ]; then
```

**Comment by maxadamo** [comment]
p.s.: now you have done some further change, and the line number is different (in my case it's 1287, in the newer version is 1291).  
ciao ciao :smile:

**Comment by matteocorti** [comment]
Thanks, I did not notice it since a long time. Released 1.93.0
